onnx_tf/backends/backend_v1.py

```diff
@@ -638,67 +639,158 @@ def handle_l_r_n(cls, node, input_dict):
+    input_size = len(node.inputs)
+    hidden_size = node.attrs["hidden_size"]
+    direction = node.attrs.get("direction", "forward")
+    num_directions = 2 if direction == "bidirectional" else 1
+
+    if node.attrs.get("output_sequence", 0) != 0:
+      raise NotImplementedError("output_sequence != 0 is not supported.")
+
+    # TODO(fumihwh): check if prev node is one of RNN
+    # process input if it comes from other previous cell
+    # which has shape [seq_length, num_directions, batch_size, hidden_size]
+    if len(input_shape) == 4 and input_shape[1] == 1:
+      x = tf.squeeze(x)
+
+    sequence_length = None
+    if input_size >= 5 and node.inputs[4] in input_dict:
+      sequence_length = input_dict[node.inputs[4]]
+
+    cell_kwargs = {}
+
+    if "clip" in node.attrs:
+      cell_kwargs["cell_clip"] = node.attrs["clip"]
+
+    tf_activations = [tf.nn.tanh]
+    if "activations" in node.attrs:
+      activations = list(map(lambda x: x.lower(), node.attrs["activations"]))
+      if activations[0] != "sigmoid":
+        raise NotImplementedError(
+            "Tensorflow uses sigmiod as first activation function `f`.")
+      if activations[1] != activations[2]:
+        raise NotImplementedError(
+            "Tensorflow uses same activation functions for `gh`.")
+      if activations[1] not in ONNX_OP_TO_TF_OP:
+        raise NotImplementedError(
+            "Activation function {} is not supported.".format(activations[1]))
+      tf_activations = [ONNX_OP_TO_TF_OP[activations[1]]]
+      if num_directions == 2:
+        if activations[3] != "sigmoid":
+          raise NotImplementedError(
+              "Tensorflow uses sigmiod as first activation function `f`.")
+        if activations[4] != activations[5]:
+          raise NotImplementedError(
+              "Tensorflow uses same activation functions for `gh`.")
+        if activations[4] not in ONNX_OP_TO_TF_OP:
+          raise NotImplementedError(
+              "Activation function {} is not supported.".format(activations[4]))
+        tf_activations.append(ONNX_OP_TO_TF_OP[activations[4]])
+
+    # TODO(fumihwh): check if reverse and bidirectional works
+    with tf.variable_scope(
+        "LSTM_" + get_unique_suffix(),
+        custom_getter=partial(custom_getter, node=node, input_dict=input_dict)):
+
+      cell_kwargs[
+          "use_peepholes"] = input_size == 8 and node.inputs[7] in input_dict
+      cell_kwargs["forget_bias"] = 0.
+      cell_kwargs["num_units"] = hidden_size
+      initial_state = None
+      initial_state_bw = None
+      if input_size >= 7 and node.inputs[5] in input_dict and node.inputs[6] in input_dict:
+        initial_state = (tf.nn.rnn_cell.LSTMStateTuple(
+            input_dict[node.inputs[6]][0], input_dict[node.inputs[5]][0]),)
+        if num_directions == 2:
+          initial_state_bw = initial_state = (tf.nn.rnn_cell.LSTMStateTuple(
+              input_dict[node.inputs[6]][1], input_dict[node.inputs[5]][1]),)
```

Comment:
Can we have more meaningful variable names than inputs[6]? 
Maybe we can extract these tensors out to variables like input_forget.